src/Entity/OgMembership.php

```diff
@@ -60,23 +60,46 @@
  *   fieldable = TRUE,
  *   bundle_entity_type = "og_membership_type",
  *   entity_keys = {
+ *     "uuid" = "uuid",
  *     "id" = "id",
  *     "bundle" = "type",
  *   },
  *   bundle_keys = {
  *     "bundle" = "type",
  *   },
  *   handlers = {
+ *     "access" = "Drupal\og\OgMembershipAccessControlHandler",
  *     "views_data" = "Drupal\og\OgMembershipViewsData",
+ *     "list_builder" = "Drupal\Core\Entity\EntityListBuilder",
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
  *     "form" = {
  *       "subscribe" = "Drupal\og\Form\GroupSubscribeForm",
  *       "unsubscribe" = "Drupal\og\Form\GroupUnsubscribeConfirmForm",
+ *       "add" = "Drupal\og\Form\OgMembershipForm",
+ *       "edit" = "Drupal\og\Form\OgMembershipForm",
+ *       "delete" = "Drupal\og\Form\OgMembershipDeleteForm",
  *     },
- *   }
+ *   },
+ *   links = {
+ *     "edit-form" = "/group/{entity_type_id}/{group}/admin/members/{og_membership}/edit",
+ *     "delete-form" = "/group/{entity_type_id}/{group}/admin/members/{og_membership}/delete",
+ *     "canonical" = "/group/{entity_type_id}/{group}/admin/members/{og_membership}/edit"
+ *   },
+ *   field_ui_base_route = "entity.og_membership_type.edit_form"
  * )
  */
 class OgMembership extends ContentEntityBase implements OgMembershipInterface {
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function urlRouteParameters($rel) {
```

Comment:
Let's prepare for #493:
```suggestion
  protected function urlRouteParameters($rel):array {
```

cc @pfrenssen

Reply:
I'm fine not expanding the scope of this with strict typing :) Let's not make it more difficult than needed.